src/app/product-list/product-list.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component } from '@angular/core';
+import { ProductComponent } from '../product/product.component';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-product-list',
+  standalone: true,
+  imports: [ProductComponent,CommonModule,FormsModule],
+  templateUrl: './product-list.component.html',
+  styleUrl: './product-list.component.css'
+})
+export class ProductListComponent {
+
+   data1 = "This is parent Component";
+
+   products = [
```

Comment:
why are you hardcoding these values ? you should be making HTTP call like you did another example using httpService